focallocal/talk
# Update with code from git@github.com:coralproject/talk.git

## What does this PR do?
Updates git@github.com:focallocal/talk.git with latest patches from git@github.com:coralproject/talk.git

## How do I test this PR?

- click a button
- view the cat
- see the cat meow

//cc @AndyatFocallocal
